packages/select/src/ListMenu/ListMenu.types.ts

```diff
@@ -0,0 +1,12 @@
+import { PopoverProps } from '@leafygreen-ui/popover';
+
+import { DropdownWidthBasis } from '../Select/Select.types';
+
+export type ListMenuProps = {
+  children: React.ReactNode;
+  id: string;
+  referenceElement: React.MutableRefObject<HTMLElement | null>;
+  className?: string;
+  labelId?: string;
+  dropdownWidthBasis: DropdownWidthBasis;
+} & Omit<PopoverProps, 'active' | 'refEl'>;
```

Comment:
Do `className` and `children` need to be explicitly defined here since I assume they're defined in `PopoverProps`? Or should they come from a `HTMLElementProps` definition?